core/libretro/vmem_utils.cpp

```diff
@@ -47,7 +47,9 @@ bool mem_region_lock(void *start, size_t len)
 {
 	size_t inpage = (uintptr_t)start & PAGE_MASK;
 	if (mprotect((u8*)start - inpage, len + inpage, PROT_READ))
-		die("mprotect failed...");
+		#ifndef IOS
```

Comment:
I don't think it's OK if this call fails on iOS. Probably it shouldn't be called at all, or the result should be checked.
This is required to detect texture updates even when using the interpreter.